src/interfaces/chain.cpp

```diff
@@ -186,6 +187,20 @@ class ChainImpl : public Chain
         }
         return nullopt;
     }
+    Optional<bool> filterMatchesAny(const uint256& hash, const GCSFilter::ElementSet& filter_set) override
```

Comment:
Please [use `std::optional` in new code](https://github.com/bitcoin/bitcoin/blob/master/src/optional.h#L12).